MericCelik/assignment4/Main.java

```diff
@@ -0,0 +1,70 @@
+public class Main {
+
+    /**Does Depth-First Search and unites current node and neighbour node into same set.
+     * @param nodetiles 2D node array to iterate.
+     * @param r row number
+     * @param c column number
+     * @param visited 2D integer array to keep track visited nodes.
```

Comment:
The `visited` array can be a boolean instead of integer type, since it always stores either zeros or ones (if not visited or if visited).

Reply:
You are right, I will fix it! :smile: